src/duckx.hpp

```diff
@@ -37,14 +40,15 @@ namespace duckx {
         bool set_text(const char *);
 
         Run &next();
-        bool has_next();
+        bool has_next() const;
```

Comment:
Could also add `const` to `get_text`, `set_text`, and `save`?

Reply:
I needed this change for the old version of the iterator. Currently I am handling this by having xml_node's in the iterator and not the duckx objects. I have not reverted the changes retroactively. I definitely think all the functions should be const if they are able.

Reply:
I'll set up a PR for `const`-ing everything that can be.

Reply:
Should be able to rebase now.

Reply:
Rebased. Thanks!